app/assets/stylesheets/all/components/_elmo_forms.scss

```diff
@@ -110,7 +110,7 @@ div.form-field {
       white-space: nowrap;
 
       i.hint {
-        display: inline-block;
+        display: inline-block !important;
```

Comment:
Yeah using `!important` is pretty much a no-no. Can you think of another way to fix this? What is causing the problem?

Reply:
I can think of another way

display is being set to `none` on click
this is because of the classes being used for certain events in the backbone file